authentication/MediaWikiAuthentication.js

```diff
@@ -94,6 +94,7 @@ MWAuthenticator.prototype = {
 			requestInfo += ' with body: ' + requestBody;
 			msg.setRequestBody(requestBody);
 		}
+		msg.getRequestHeader().setContentLength(requestBody.length);
```

Comment:
Better with `msg.getRequestBody().length()` (avoids byte vs char issues and good example for other scripts).

Reply:
Ok I'll tweak that.